alfresco-transform-pdf-renderer/alfresco-transform-pdf-renderer-boot/src/main/resources/application-default.yaml

```diff
@@ -1,5 +1,9 @@
 queue:
   engineRequestQueue: ${TRANSFORM_ENGINE_REQUEST_QUEUE:org.alfresco.transform.engine.alfresco-pdf-renderer.acs}
 transform:
-  config:
-    location: classpath:pdfrenderer_engine_config.json
\ No newline at end of file
+  core:
+    config:
+      location: classpath:pdfrenderer_engine_config.json
+    pdfrenderer:
+      executor:
+        path: ${PDF_RENDERER_EXE:/usr/bin/alfresco-pdf-renderer}
```

Comment:
For consistency (with ImageMagick) why not: remove "executor" and rename "path" to "exe" ?

```
transform:
  core:
    config:
      location: classpath:pdfrenderer_engine_config.json
    pdfrenderer:
      exe: ${PDFRENDERER_EXE:/usr/bin/alfresco-pdf-renderer}
```

Reply:
Note: above I've also removed the extra underscore between PDF & RENDERER which potentially can confusion with Spring Boot mapping from dot notation.